libs/wire-api/src/Wire/API/Routes/Version.hs

```diff
@@ -204,6 +207,9 @@ isDevelopmentVersion _ = True
 developmentVersions :: [Version]
 developmentVersions = filter isDevelopmentVersion supportedVersions
 
+toDisabledVersions :: Maybe Bool -> Set Version
+toDisabledVersions mEnableDevAPI = Set.fromList $ maybe developmentVersions (const []) mEnableDevAPI
```

Comment:
Wouldn't it be better to check the `Bool`? With Helm in mind, it might be okay to assume that a set value also is true, but there might be different deployment methods than Helm. I could imagine that an on-prem admin manually changes the value from `true` to `false` or (manually and) explicitly wants to set it to `false`. In these cases, this logic would still assume `true`.

Reply:
You are right, I missed it.